plugin.video.openingsmoe/main.py

```diff
@@ -0,0 +1,22 @@
+import sys
+from urllib import urlencode
+from urlparse import parse_qsl
+import xbmcgui
+import xbmcplugin
+import xbmc
+import json,urllib
+
+
+
+def playVideo(jslist):
+    pl = xbmc.PlayList(xbmc.PLAYLIST_VIDEO)
+    pl.clear()
+    for u in jslist:
+        pl.add("http://openings.moe/video/"+u['file'], xbmcgui.ListItem(u['source']+" "+u['title']))
+    pl.shuffle()
+    xbmc.Player().play(pl, windowed=False)
```

Comment:
plugins should not use `xbmc.Player()` (plugins only set the resolved url on listitems). Custom player instances should be only used in scripts